### ISSUE TITLE
Problems with Spotify

### ISSUE THREAD

**Issue by XyceL** [AllToMP3/alltomp3-app#150]
I have a problem... I was downloading some songs.. but, it stopped and it doesn't work. Why?

**Comment by bzsimon** [comment]
Yep, it seems not dwnloading any playlist from spotify .... waiting for a new version :-) maybe will solve the problem.

**Comment by coltc50** [comment]
It worked for me for 3 albums and now it is stuck and I can't make it work anymore... Tried to reinstall but nothing helps...

**Comment by julmot** [comment]
Same problem, since a couple months already.

**Comment by AurelienConil** [comment]
Same for me, download does not start.

**Comment by Ayoamaro** [comment]
I can't download music from Spotify or other plataforms. Solutions?

**Comment by David-H-1** [comment]
Same problem for me. Can do nothing but abort. I had waited for 30 minutes.

**Comment by sts-ryan-holton** [comment]
Same for me, any fix?

**Comment by nuganics** [comment]
I'm using Sidify and it is working.

**Comment by AmazingGabriel16** [comment]
By the way, in case people cannot install Sidify as a temporary replacement as the dev fixes AllToMP3, use OBS as Sidify is simply a recorder.
